internal/resource/common/id/id_schema.go

```diff
@@ -0,0 +1,40 @@
+package id
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/resource/configvalidators"
+)
+
+// Get schema elements common to all resources
+func Schema(s *schema.Schema) {
```

Comment:
I'd name these methods something like AddIdSchema or whatever to imply they are adding to the schema object being passed in